tests/integration/vrf_test.yaml

```diff
@@ -0,0 +1,368 @@
+description: Delete the vrf interface test-vrf0
+interfaces:
+- name: test-vrf0
+  type: vrf
+  state: absent
+description: Create the vrf interface test-vrf0 with vrf port eth1 and eth2 attached
+interfaces:
+- name: test-vrf0
+  type: vrf
+  state: up
+  vrf:
+    port:
+    - eth1
+    - eth2
+description: Create the vrf interface test-vrf0 with vrf port eth1 and vrf route table ID 100
```

Comment:
There is not table ID for this YAML.

Reply:
The diff state is generated by the test `test_add_and_remove_port`, where the vrf table id 100 is indeed specified, however, the previous test is `test_create_with_empty_ports`, and the vrf table id 100 is applied but there is no clean up in that function.

Reply:
I added a commit to this PR for fixing the issue.